### PR TITLE
[FIX] l10n_ar_edi_ux: cambiar string del campo dst_country de "País Origen" a "País de destino de la mercadería"

### PR DESCRIPTION
**Pull request by pablohmontenegro**
Ticket: 74144

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/odoo-argentina-ee/pull/373).

File: l10n_ar_edi_ux/models/l10n_ar_boarding_permission.py [hunk 1]
```diff
@@ -25,3 +30,16 @@ def name_search(self, name, args=None, operator='ilike', limit=100):
...
+        """ No queremos que existan registros repetidos de permisos de embarque. """
+        if (
+            self.search_count(
+                [("number", "=", self.number), ("dst_country", "=", self.dst_country.id), ("company_id", "=", self.company_id.id)]
+            )
+            > 1
+        ):
+            raise ValidationError(
+                ("Error! El permiso de embarque ya existe.")
+            )
```
Comment (zaoral) [inline l10n_ar_edi_ux/models/l10n_ar_boarding_permission.py]: delete this, leace only the sql constraint. also add a pre script migration to resolve the problem of duplciated records

**Review by zaoral — CHANGES_REQUESTED** [review]

File: l10n_ar_edi_ux/models/l10n_ar_boarding_permission.py [hunk 2]
```diff
@@ -6,9 +6,13 @@ class L10nArBoardingPermission(models.Model):
     _description = 'Boarding Permission'
 
     number = fields.Char(string="Permiso de embarque", required=True, size=16)
-    dst_country = fields.Many2one('res.country', string="País Origen", help="País donde se aplica permiso de embarque", required=True)
+    dst_country = fields.Many2one('res.country', string="País de destino", help="País de destino de la mercadería", required=True)
```
Comment (zaoral) [inline l10n_ar_edi_ux/models/l10n_ar_boarding_permission.py:9]: delete line

File: l10n_ar_edi_ux/__manifest__.py [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
     'name': 'Argentinian Electronic Invoicing UX',
-    'version': "16.0.1.7.0",
+    'version': "16.0.1.8.0",
```
Comment (zaoral) [inline l10n_ar_edi_ux/__manifest__.py]: rebase, and change manifest version

**Review by zaoral — CHANGES_REQUESTED** [review]

**Review by zaoral — APPROVED** [review]

**Comment by zaoral** [comment]
@roboadhoc r+

**Comment by fw-bot-adhoc** [comment]
@pablohmontenegro @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/odoo-argentina-ee#380

**Comment by fw-bot-adhoc** [comment]
@pablohmontenegro @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/odoo-argentina-ee#380

**Comment by fw-bot-adhoc** [comment]
@pablohmontenegro @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/odoo-argentina-ee#380

**Pull request closed without merging**